### PR TITLE
Fix/40

### PR DESCRIPTION
**Pull request by Friedrich-Mocker**
changed the fadeIn animation from being jquery based to css based
iptools.jquery-modal- 40
https://github.com/interactive-pioneers/iptools-jquery-modal/issues/40

### CODE REVIEW

**Comment by ain** [comment]
@beroes please review.

**Comment by ain** [comment]
@Friedrich-Mocker please keep to standard commit messages as documented in IP Conventions shared earlier. Thanks.

**Comment by Friedrich-Mocker** [comment]
i forgot to fix the unit tests.

**Comment by ain** [comment]
Yes, also if you're implementing a new feauture somewhere, please add a test for it. Will serve us/you well later :)

**Comment by Friedrich-Mocker** [comment]
Please review the new and changed unit tests in depth, because i have litte experience wriing unit tests in jasmine :)

File: src/iptools-jquery-modal.js [hunk 1]
```diff
@@ -32,7 +32,9 @@
         closeButton: '__button-close',
         spinner: '__spinner',
         content: '__content'
-      }
+      },
+      overlayFadeIn: 'overlay--fade-in',
+      overlayFadeOut: 'overlay--fade-out'
```
Comment (beroes) [inline src/iptools-jquery-modal.js]: please use a correct indentation here

**Comment by ain** [comment]
@Friedrich-Mocker first things first. The build is failing over SCSS formatting (see Travis build log).

**Comment by Friedrich-Mocker** [comment]
iIl have to fix my local scss linter. it doesnt recognize this.

**Comment by ain** [comment]
@Friedrich-Mocker please also confirm with @GeorgMeyer23 if the IP Conventions page is linked to up to date SCSS lint configuration.

**Comment by beroes** [comment]
Checked it out, tested, it works.

File: src/iptools-jquery-modal.js [hunk 2]
```diff
@@ -228,7 +232,8 @@
 
     function hideOverlay() {
       if ($overlay) {
-        $overlay.stop().fadeOut(settings.animationDuration);
+        $overlay.removeClass(settings.overlayFadeInClass);
+        $overlay.addClass(settings.overlayFadeOutClass);
```
Comment (ain) [inline src/iptools-jquery-modal.js]: You can chain here to save time, both work with `$overlay`.

File: src/iptools-jquery-modal.scss [hunk 3]
```diff
@@ -11,11 +11,25 @@ $IPTModal_animation-speed: 0.25s;
...
+    opacity: 1;
+
+    pointer-events: all;
+  }
+
+  &--fade-out {
+    opacity: 0;
+
+    pointer-events: none;
+  }
```
Comment (ain) [inline src/iptools-jquery-modal.scss]: BEM nicely kept, thanks 👍

**Comment by ain** [comment]
Needs rebase.

**Comment by ain** [comment]
Rebased to #70. Closing.

**Pull request closed without merging**